sycl/doc/extensions/supported/sycl_ext_oneapi_backend_level_zero.md

```diff
@@ -135,9 +146,35 @@ struct {
 }
 ```
 
-Supported since version 3 of this specification.[^1]
+Starting in version 3 of this specification.
 </td>
-</tr><tr>
+</tr>
+<td rowspan="1">
+
+``` C++
+std::variant<ze_command_queue_handle_t,
+             ze_command_list_handle_t>
+```
+</td>
+<td>
+
+``` C++
+struct {
+  std::variant<ze_command_queue_handle_t,

```

Comment:
Have we considered making this struct template instead of using `std::variant`?

Reply:
I don't think this is possible.  If we do this, then we need to change `backend_input_t` to add another template parameter.  Since that type is defined in the core SYCL specification, we can't change the number of template parameters.